core/src/test/java/io/questdb/griffin/EngineMigrationTest.java

```diff
@@ -119,6 +117,12 @@ public void test425() throws IOException, SqlException {
         doMigration("/migration/data_425.zip", true, true, true, true);
     }
 
+    @Test
+    public void test426() throws IOException, SqlException {
+        Assume.assumeTrue(Os.type != Os.WINDOWS); // the zip file is not ok with windows, it was created in a mac
```

Comment:
it should run on windows. Comment is misleading.

The reason it doesn't run on windows is that `data_426.zip` contains `_tab_index.d`. Zip file cannot be unzipped because this file is owned (open) by running `CairoEngine`. Remove this file inside the archive

Reply:
cool, that is nice to know thank you. Soon I will be able to work these things out myself with my new supper dupper windows box. Thank you for the explanation.